### PR TITLE
docs: optimize the introduction docs

### PR DESCRIPTION
**Pull request by YeSuX**

### CODE REVIEW

**Comment by productdevbook** [comment]
Is this over ?

**Comment by YeSuX** [comment]
> Is this over ?

That's all for now for introductions. Because I tried two examples, there were more or less a few problems.

I want to write other sections of the document first, while writing the corresponding problems will be fixed. Finally, integrate the examples in the summary into introduce and examples.

File: .docs/content/motion/1.getting-started/1.introduction.md [hunk 1]
```diff
@@ -14,8 +14,67 @@ In this quick overview, we'll take a look at some of the APIs that Oku Motion of
...
+      repeat: Infinity,
+      repeatDelay: 1,
+    }"
+  />
+```
+
+<!-- <iframe
+  src="https://stackblitz.com/edit/vitejs-vite-bqgiwz?embed=1&file=src%2FDemo.vue&hideExplorer=1&hideNavigation=1&view=preview"
+  style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
+></iframe> -->
```
Comment (productdevbook) [inline .docs/content/motion/1.getting-started/1.introduction.md:70]: why comment ?

Comment (YeSuX) [reply .docs/content/motion/1.getting-started/1.introduction.md:70]: This is a bug of animation.

https://github.com/oku-ui/motion/issues/8

When the bug is fixed,I will add it.

**Comment by productdevbook** [comment]
Thank you ❤️

**Pull request merged**